src/app/routes/onDemandRadio/getInitialData/index.test.js

```diff
@@ -22,6 +22,9 @@ describe('Get initial data for on demand radio', () => {
     expect(pageData.summary).toEqual('د بي بي سي ورلډ سروس څخه پروګرام کول');
     expect(pageData.language).toEqual('ps');
     expect(pageData.metadata.type).toEqual('On Demand Radio');
+    expect(pageData.imageUrl).toEqual(
+      'ichef.bbci.co.uk/images/ic/$recipe/p063j1dv.jpg',
```

Comment:
Do we need to replace the $recipe placeholder?

Reply:
Yes we do. This will be replaced later.

Reply:
Samora introduced using the `getPlaceholderImageUrlUtil` into the `getInitialData` file for the schema.org thumbnail, should we be passing this `imageUrl` into it as well? (at which point - both `imageUrl` & `thumbnailImageUrl`  would become the same value)

https://github.com/bbc/simorgh/pull/6471/files/b6c49aa9866e5ea83aa06964d807e3cc201d3a81#diff-b5e90daa71070c1688f08e67391dfef3R48-R49

Reply:
The `getPlaceholderImageUrlUtil` works differently from what we require. The heading image comes at different sizes for different views and we use `srcset` to achieve that. We will have to write two util functions to handle it the routes level and I think that's an overkill